dbmigrate-lib/Cargo.toml

```diff
@@ -12,10 +12,16 @@ keywords = ["database", "postgres", "migration", "sql", "mysql"]
 [dependencies]
 regex = "0.2"
 url = "1"
-postgres = { version = "0.14", features = ["with-native-tls"] }
-mysql = "9.0"
-rusqlite = "0.9"
+postgres = { version = "0.14", features = ["with-native-tls"], optional=true }
+mysql = { version="9.0", optional=true}
+rusqlite = { version="0.9", optional=true}
 error-chain = "0.10"
 
 [dev-dependencies]
 tempdir = "0.3.4"
+
+[features]
+default = ["postgres_support", "sqlite_support", "mysql_support"]
+postgres_support = ["postgres"]
+sqlite_support = ["rusqlite"]
+mysql_support = ["mysql"]
```

Comment:
When I try that I get an error -

```
Caused by:
  Features and dependencies cannot have the same name: `postgres`
```

Presumably due to the dependency on the `rust-postgres`.  I agree that `*_support` is pretty verbose though - can you think of a better feature name?